setup.py

```diff
@@ -23,13 +23,16 @@
         'symmetrical encryption of cached values using the python cryptography library'
     ),
     url='https://github.com/foundertherapy/django-redis-secure/',
-    download_url='https://github.com/foundertherapy/django-redis-secure/archive/1.0.0.tar.gz',
+    download_url='https://github.com/foundertherapy/django-redis-secure/archive/1.0.1.tar.gz',
     author='Ghaleb Khaled',
     author_email='ghaleb@foundertherapy.co',
     install_requires=[
-        'Django>=1.9.6',
+        'Django==1.9.6',
         'cryptography>=1.4',
         'django-redis==4.4.3',
+        'django-rq==0.9.1',
```

Comment:
@GhalebKhaled Our library is going to be different and independent from `django-rq`, we can actually act as if we have our own version of it so using a minimum minor version will not affect us, I think.